### ISSUE TITLE
Identifier "kgml_Script_anon_40_1316_40...." has already been declared

### ISSUE THREAD

**Issue by Grisgram** [YoYoGames/GameMaker-Bugs#8316]
### Description
Hello there,

Russell reopened #6596 after my report on discord, that the current beta brings back the "already been declared" bug, but he couldn't reproduce it.

The attached project here shows the bug. This is an (almost finished) commercial game, please treat very confidential.
You can't pass through the server login but you don't need to, because the game doesn't even start due to the error above.

### Steps To Reproduce
Just open the attached project, set "HTML5/JavaScript" and configuration `Default` as target and Press F5.

You see this in the console of the browser:

```
Uncaught SyntaxError: Identifier 'kgml_Script_anon_40_1316_40_anon_40_1180_40_GuiMouseTranslator_40_GuiMouseTranslator' has already been declared

index.html:85 Uncaught ReferenceError: GameMaker_Init is not defined
    at index.html:85:27
```

### Which version of GameMaker are you reporting this issue for?
IDE v2024.1100.0.665 Runtime v2024.1100.0.691

### Which operating system(s) are you seeing the problem on?
Windows 10.0.19045.0

### Which platform(s) are you seeing the problem on?
HTML5

[fab2dfc0-1930-42b3-8890-4265fb9c6faf](https://api.gamemaker.io/api/github/downloads/fab2dfc0-1930-42b3-8890-4265fb9c6faf)

**Comment by rwkay** [comment]
Fixed in 2024.11

**Comment by cameron-home** [comment]
Verified fixed in IDE v2024.1100.0.687  Runtime v2024.1100.0.708

**Comment by Grisgram** [comment]
Yes, verified from customer side also - we have a working html publish of our system. Thanks again!
